packages/ui-extensions/docs/surfaces/admin/staticPages/examples/direct-api-fetch.jsx

```diff
@@ -19,22 +20,14 @@ async function getProduct(id) {
       variables: {id},
     }),
   });
-  return res.json();
+  const {data} = await res.json();
+  return data.product;
 }
 
-function Extension() {
-  // Contextual "input" data passed to this extension:
-  const {data} = shopify;
-
-  const [product, setProduct] = useState();
-  useEffect(() => {
-    const productId = data.selected?.[0]?.id;
-    getProduct(productId).then(({data}) => setProduct(data.product));
-  }, [data]);
-
+function Extension({product}) {
   return (
-    <s-admin-block title="Product Info">
-      <s-text>The selected product title is {product?.title}</s-text>
+    <s-admin-block heading="Product Info">
```

Comment:
`s-admin-block` has `heading`, not `title`. 

Source: https://shopify.dev/docs/api/admin-extensions/2025-10-rc/polaris-web-components/other/adminblock